### PR TITLE
KDE .svg monochrome fix

### PR DESCRIPTION
**Pull request by now-im**

### CODE REVIEW

**Comment by keeferrourke** [comment]
@now-im can you run [`svgcleaner`](https://github.com/RazrFalcon/svgcleaner) over these files to reduce their size and make sure they still work before I merge these changes?

From now on, I'm going to be running `find ~/.icons/la-captaine -type f -name "*.svg" -exec svgcleaner {} {} \;` to make sure that La Capitaine stays a reasonable size.

**Comment by now-im** [comment]
SVG cleaning does break the standard KDE is provided by this link:
https://kdeonlinux.wordpress.com/2016/04/25/performance-update-for-breeze-icons/

**Comment by now-im** [comment]
@keeferrourke Is all symbolic icon follows this rule:
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">`
Rather than this:
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22">`

**Comment by keeferrourke** [comment]
Yes, symbolic icons are usually 16x16. They should always be under a `symbolic/` directory and named `iconname-symbolic.svg`. Do not recolour or add styling to symbolic icons, as they have absolutely nothing to do with KDE.

**Comment by now-im** [comment]
I have made changes according to the guidelines of @DarkknightAK and I didn't change the color or style. I made a mistake by editing them as 22x22. I have fixed it. Let me give you an example:
Before:
`<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="m7.0222143 2-4.2857143 12h2.1428571l.9107143-2.571429h4.1785715l.9107141 2.571429h2.383929l-4.526786-12zm .8571428 3.4285714 1.5 4.2857146h-3z" fill="#bebebe"/></svg>`
After:
```
svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22">
  <defs id="defs3051">
    <style type="text/css" id="current-color-scheme">
      .ColorScheme-Text {
        color:#bebebe;
      }
      </style>
  </defs>
    <path
       style="fill:currentColor;fill-opacity:1;stroke:none"
       d="m7.0222143 2-4.2857143 12h2.1428571l.9107143-2.571429h4.1785715l.9107141 2.571429h2.383929l-4.526786-12zm .8571428 3.4285714 1.5 4.2857146h-3z"
       class="ColorScheme-Text" />
 </svg>
```
Besides, @demoy listed scalable .svg, so I edited  them. After every edit, I preview the file so any noticable change and I list them as icons I could not fix.

**Comment by keeferrourke** [comment]
@now-im, I really appreciate your work and your help tackling this issue! I think you misunderstand me however. The defs block (as follows) is not needed for `*-symbolic.svg` files and unfortunately adds additional bulk to the project. Symbolic icons are only used by GNOME apps and toolkits. Please undo any changes you have made to symbolic icons, since KDE does not use them and GNOME does not make use of CSS styling.
```
 <defs id="defs3051">
    <style type="text/css" id="current-color-scheme">
      .ColorScheme-Text {
        color:#bebebe;
      }
      </style>
  </defs>
```

At the very least before I can accept these changes, these icons need to be "minified" by removing newlines and extraneous spacing. Please run the [svgcleaner](https://github.com/RazrFalcon/svgcleaner) utility on each file you edit.

**Comment by now-im** [comment]
I have updated my system with all the recent symbolic changes. This is what my system gained.

```
Total Installed Size:  110.98 MiB
Net Upgrade Size:        0.02 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                  [#############################################] 100%
(1/1) checking package integrity                                                [#############################################] 100%
(1/1) loading package files                                                     [#############################################] 100%
(1/1) checking for file conflicts                                               [#############################################] 100%
(1/1) checking available disk space                                             [#############################################] 100%
:: Processing package changes...
(1/1) upgrading la-capitaine-icon-theme-git                                     [#############################################] 100%
:: Running post-transaction hooks...
(1/2) Updating icon theme caches...
(2/2) Arming ConditionNeedsUpdate...
```
And I have written it before that .svg cleaning nullifies all my changes and restores the icons it to the previous state. Now if you still want me to restore the symbolicfiles  I will do it.

**Comment by now-im** [comment]
I believe my work here is done and you can merge the commits as it doesn't contain any symbolic changes anymore.

**Comment by keeferrourke** [comment]
I will merge this tomorrow or thursday, since these icons still need to be cleaned by `svgcleaner`. Ping me if I forget.

**Comment by now-im** [comment]
Hold this. I have some time. I intend to recode every action/light/22x22 icon. Will take some time but it will definitely fix the problem.

**Comment by now-im** [comment]
List of Icons that I don't have enough knowledge to fix:
Folder: /la-capitaine-icon-theme/actions/22x22-light/
List:

- [ ] align-*.svg

- [ ] view-media-playlist.svg (Important)

- [ ] antivignetting.svg

**Comment by keeferrourke** [comment]
@now-im if you just keep track of a list of icons you don't know how to fix, I should be able to take care of them. Thank you so much for helping to make this clean-up possible &mdash; I wouldn't be able to tackle this issue myself.

**Comment by now-im** [comment]
I am doing that. The icons in the list are way more complex and I can not solve them.  The icons that pop up everywhere, I am marking them as important.

File: actions/22x22-light/.directory [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[Dolphin]
```
Comment (keeferrourke) [inline actions/22x22-light/.directory]: Please do not commit `.directory` files, or other extraneous filesystem metadata.

Comment (keeferrourke) [reply actions/22x22-light/.directory]: Any `.directory` files you have committed will need to be removed

**Comment by now-im** [comment]
Is the problem fixed now?

**Comment by now-im** [comment]
I got a question:

How to turn this:
`<svg viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg"><g fill="#4d4d4d" transform="matrix(0 1 -1 0 1 -1031.3622)"><rect height="13.000001" ry=".5" width="6.000004" x="1034.3622" y="-18"/><rect height="6" ry=".5" width="6.039805" x="1044.3622" y="-11"/><rect height="1.5" ry=".5" width="15.999797" x="1034.3624" y="-3.5"/></g></svg>`
Into something like this:
`<svg viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg"><path d="m16 3v2h-2v1h2v2h1v-2h2v-1h-2v-2zm-8.0898438 2-3.3144531 9.59375-.1230469.40625-1.4726562 4h1.6582031l1.625-4.4375h5.4335939l1.625 4.4375h1.658203l-1.472656-4-.123047-.40625-3.314453-9.59375h-.214844-1.75zm1.1054688 1.9375 2.179687 6.34375h-4.3906245z" fill="#4d4d4d"/></svg>`
With this I can fix everything. 
On an important note, I think you should merge up to this point and see how the fix works. I will be away from the computer up to 2 weeks. So the rest of the fix will take some time.

**Comment by keeferrourke** [comment]
@now-im which files are those which you pasted in the above comment?

I'll refrain from merging until I get a chance to review properly, which may also not be for a few weeks. I have a lot going on for school right now, so I hope you understand.

**Comment by now-im** [comment]
Okay, Got it as you wish. 
For example:
This is from `align-horizontal-left-out.svg`:
`<svg viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg"><g fill="#4d4d4d" transform="matrix(0 1 -1 0 1 -1031.3622)"><rect height="8" ry=".5" width="6.000038" x="1034.3622" y="-18"/><rect height="6.03614" ry=".5" width="6.03614" x="1044.3622" y="-18"/><rect height="3" ry=".5" width="5.999982" x="1044.3622" y="-5"/><rect height="1.5" ry=".5" width="16" x="1034.3622" y="-8.5"/></g></svg>`
Which I can not edit, because it does not contain any `d=***` info. like this:
`<svg viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg"><path d="m4 3c-.55402 0-1 .44598-1 1v14c0 .55402.44598 1 1 1h14c .55402 0 1-.44598 1-1v-14c0-.55402-.44598-1-1-1zm7 3c .90721 0 1.7436.22283 2.50977.66992.76601.44709 1.37322 1.0543 1.82031 1.82031.44693.76601.66992 1.60264.66992 2.50977 0 .90705-.22291 1.74368-.66992 2.50977-.44709.766-1.0543 1.3733-1.82031 1.82031-.76601.44709-1.60264.66992-2.50977.66992s-1.74368-.22283-2.50977-.66992-1.37322-1.05438-1.82031-1.82031c-.44701-.76617-.66992-1.60264-.66992-2.50977 0-.90705.22275-1.74368.66992-2.50977.44709-.76609 1.05422-1.3733 1.82031-1.82031.76601-.44709 1.60264-.66992 2.50977-.66992zm-1.4707 2.51953c-.11723 0-.21832.0406-.30078.12305l-.58594.58594c-.0825.0825-.12305.18355-.12305.30078 0 .11283.0406.21058.12305.29297l1.17969 1.17773-1.17969 1.17773c-.0825.0824-.12305.18014-.12305.29297 0 .11715.0406.2184.12305.30078l.58594.58594c.0825.0825.18355.12305.30078.12305.11283 0 .2105-.0406.29297-.12305l1.17773-1.17969 1.17773 1.17969c.0823.0825.18014.12305.29297.12305.11723 0 .21832-.0406.30078-.12305l.58594-.58594c.0825-.0824.12305-.18363.12305-.30078 0-.11283-.0406-.21058-.12305-.29297l-1.17969-1.17773 1.17969-1.17773c.0825-.0824.12305-.18014.12305-.29297 0-.11723-.0406-.21832-.12305-.30078l-.58594-.58594c-.0825-.0825-.18355-.12305-.30078-.12305-.11283 0-.21066.0406-.29297.12305l-1.17773 1.17969-1.17773-1.17969c-.0825-.0825-.18014-.12305-.29297-.12305z" fill="#4d4d4d"/></svg>`
It is from `view-time-schedule-baselined-remove.svg`.

**Comment by now-im** [comment]
After a very tiring editing process, finally, it is done. There are only couple of files I could not fix and I have listed them above. My work will pay off when I will see the fix in my KDE system. Good luck with the future development! :+1:

**Comment by now-im** [comment]
I don't know whether to be pleased or angry. I was suggested that I should edit only 22*22-light folder now, it seems the problem lies within the dark folder. Luckily I am in the Linux world and command line can solve anything. It took quite a few hours to get the dark folder fixed. Pushing the commit now. The result is heartwarming though. 
spectacle tj6028

**Comment by keeferrourke** [comment]
@now-im thank you so much for all your help with this. I hope to merge these changes by Tuesday next week. You'll be getting a huge shout out in the  release notes for the next version and more immediately a mention in the [`Thanks.md`](https://github.com/keeferrourke/la-capitaine-icon-theme/blob/master/Thanks.md) document. I would never have been able to do this myself.

**Comment by keeferrourke** [comment]
@now-im some of the icons which you committed here are corrupt. Namely, document-save (a pretty important one!) was invalid. I've fixed it, but it's likely there are others if you manually edited the files instead of using some sort of regex/replace... Can you go through and try to identify any other icons which may be corrupt?

**Comment by now-im** [comment]
I am in some sort of family crisis right now. I will do it. In my kde i have not stumbled upon any sort of problen yet.

**Comment by now-im** [comment]
I am in some sort of family crisis right now. I will try do it but can not promise now. In my kde i have not stumbled upon any sort of corrupted icon problem yet.

**Comment by keeferrourke** [comment]
Yikes. Best of luck and no worries.

**Comment by now-im** [comment]
What is the problem? In my old code, the icons are opening properly.
spectacle tj1758
spectacle tj1851

**Comment by keeferrourke** [comment]
Inkscape probably opens them fine, but `eog` could not display them, and GTK2+ applications would show "missing icon" icons.

**Comment by now-im** [comment]
kindly make a list of them. I will try to edit them by your standard. The list should not be long.

**Comment by now-im** [comment]
In my files, the format is:

- style, d, class, transform
In your files, the format is:

- style, class, d, transform
That is the difference I am seeing.

**Comment by keeferrourke** [comment]
@now-im don't worry about it, I'll fix any icons that appear broken as I come across them. Thanks for your help with tackling this issue. I'm going to lock discussion here, so that any other issues re: this PR should directed to a new issue and handled accordingly.

**Pull request merged**